conf/pf.conf.defaults

```diff
@@ -1155,6 +1155,11 @@ normalize_radius_machine_auth_username=disabled
 # Which attributes to use to get the username from a RADIUS request
 # The order of the attributes are listed in this configuration parameter is followed while performing the lookup
 username_attributes=PacketFence-UserNameAttribute,TLS-Client-Cert-Subject-Alt-Name-Upn,TLS-Client-Cert-Common-Name,User-Name
+#
+# radius_configuration.forward_key_balanced
+#
+# Send in the proxy request the attribute PacketFence-KeyBalanced to randomize the load balance in the next PacketFence RADIUS server
+PacketFence-KeyBalanced=disabled
```

Comment:
That parameter name doesn't follow the convention in pf.conf.defaults

And also, not the same name as in documentation.conf (which is fine there)